### ISSUE TITLE
Bug Report: Taking a screenshot while the sticker transition is happening causes the screen to have a white effect for the rest of the game.

### ISSUE THREAD

**Issue by Prod-42** [FunkinCrew/Funkin#2811]
## Describe the bug

Taking a screenshot while the stickers transition is happening causes the game to have a white effect over the screen for the rest of the game, the only way to remove it is to restart the game.

## To Reproduce

1. Start any song
2. Exit to main menu
3. While the sticker transition is happening take a screenshot before it ends
4. The game should have the white effect now

## Expected behavior

The game looking the same as before without that effect.

## Screenshots/Video

https://github.com/FunkinCrew/Funkin/assets/85413903/5b289672-103a-4a11-afa8-5ce7d7237765

![screenshot-2024-06-14-02-57-31](https://github.com/FunkinCrew/Funkin/assets/85413903/848eea94-3c87-419d-8e66-8ca874d346a3)

![screenshot-2024-06-14-02-57-41](https://github.com/FunkinCrew/Funkin/assets/85413903/4883efa9-0ede-4246-a3c2-896c4606a3f0)

## Desktop
 - OS: Windows 11
 
 - Browser: Nope
 
 - Version: 0.4.1
 

## Additional context

While trying to replicate this bug it's probable that the screenshot preview stays in there too for the rest of the game, I know Eric is aware of this from one of his streams but it's just an extra.

**Comment by Prod-42** [comment]
It seems like this got fixed in the update!

I'm closing this issue now.

**Comment by Prod-42** [comment]
Nevermind it happened again.

Same steps too

https://github.com/user-attachments/assets/beeafda2-6292-475b-86ca-cf7abbcc0408

**Comment by Lasercar** [comment]
Does this happen with https://github.com/FunkinCrew/Funkin/pull/4082?

**Comment by Lasercar** [comment]
Duplicate of https://github.com/FunkinCrew/Funkin/issues/2284

**Comment by Hundrec** [comment]
I'll keep this one separate from the screenshot preview bug

**Comment by Prod-42** [comment]
Yeah this issue is related to #2284 but it's not exactly the same

**Comment by Lasercar** [comment]
Now fixed by my improved screenshot plugin!
